README.md

```diff
@@ -104,28 +104,27 @@ Now you are ready to do your first build and deploy.
 
 ### Creating a package
 
-As it's noted in [erllambda docs](https://github.com/alertlogic/erllambda#openssl-version) it's
-advised to package Erlang release with ERTS which is built in an
-environment that is very close to AWS Lambda environment.
+As noted in [erllambda docs](https://github.com/alertlogic/erllambda#openssl-version) it's
+advised to package the Erlang release with an ERTS built in an
+environment that is very close to AWS Lambda environment.  There are two ways to get a proper ERTS: use a premade ERTS from a docker image or include a custom ERTS.
```

Comment:
The section on getting the ERTS was pretty unclear to me, so I restructured it a bit.

* I explicitly suggested the docker image method.
* I collapsed the smallest headers into a list of actions so that they made it clear that the copy-in-an-ERTS option is a single section.
* I renamed the headers a bit